common/gnome-shell/3.36/sass/_common.scss

```diff
@@ -822,14 +822,20 @@ StScrollBar {
     border: none;
 
     .panel-button {
```

Comment:
The hovered panel clock/calendar menu button still has the black bottom border in GDM. The transparent border rule has to be applied to `.panel-button.clock-display .clock` as well, e.g. something like the following:
```
.panel-button,
.panel-button.clock-display .clock {
```

Could you also fix this while you're at it? :)

Reply:
Thanks, hadn't noticed that - done